tests/test_exiftool_direct.py

```
@@ -0,0 +1,79 @@
+            print(f"   ERREUR: {e.stderr}")
+        
+        # Lire résultat
+        result = subprocess.run([
+            "exiftool", "-json", str(media_path)
+        ], capture_output=True, text=True, check=True)
+        data = json.loads(result.stdout)[0]
+        print(f"   Keywords après -= +=: {data.get('Keywords', 'AUCUN')}")
+        
+        # 3. Essayer juste +=
+        print("\n3. Test avec += seulement...")
+        subprocess.run([
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)